USER_MANUAL.md

```diff
@@ -28,6 +28,12 @@ Email: <sally@highwoodemissions.com>
   - [5. Simulation Settings](#5-simulation-settings)
     - [\<parameter\_level\> (simulation\_settings)](#parameter_level-simulation_settings)
     - [\<version\> (simulation\_settings)](#version-simulation_settings)
+    - [\<n\_processes\>](#n_processes)
```

Comment:
n_processes and n_simulations will be renamed soon

Reply:
Not necessarily something wrong with your work, but I'll give you the future names to use instead so we can avoid conflicts.

Reply:
We are moving to processes_count and simulation_counts, pending review

Reply:
Regarding the rename change, I think it should be better off being in its own PR. But good to know.